index.html

```diff
@@ -0,0 +1,366 @@
+          <div
+            class="row row-cols-xl-auto row-cols-lg-2 row-cols-md-3 row-cols-sm-1 row-cols-1 justify-content-center feature-page"
+          >
+            <div
+              class="d-flex mt-2 justify-content-center align-items-center d-lg-none d-md-none"
+            >
+              <button
+                type="button"
+                class="btn btn-light btn-outline-secondary px-5 showMore"
+              >
+                More
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="14"
+                  height="14"
+                  viewBox="0 0 24 24"
+                  fill="#FF4F38"
+                >
+                  <path
+                    d="M0 7.33l2.829-2.83 9.175 9.339 9.167-9.339 2.829 2.83-11.996 12.17z"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="container-fluid footer-section bg-dark d-none d-sm-block d-md-block d-lg-block">
+      <div
+        class="container d-flex flex-column justify-content-center align-items-center"
+      >
+        <h2 class="text-center text-light pt-3">Partners</h2>
+        <div class="footer-line border-bottom border-5 border-danger"></div>
+      </div>
+      <div
+        class="container mt-lg-5 pt-lg-4 mt-md-4 pt-sm-3 mt-md-4 mt-3 pt-3 pt-sm-3 ps-5 d-flex justify-content-center align-items-center gap-lg-5 gap-md-3 gap-sm-2"
+      >
+        <div class="">
+          <img
+            class="img-fluid partners"
+            src="./assets/imgs/mcdonalds.png"
+            alt=""
+          />
+        </div>
+        <div class="">
+          <img
+            class="img-fluid partners"
+            src="./assets/imgs/fries.png"
+            alt=""
+          />
+        </div>
+        <div class="">
+          <img
+            class="img-fluid partners"
+            src="./assets/imgs/nestle.png"
+            alt=""
+          />
+        </div>
+        <div class="">
+          <img
+            class="img-fluid partners"
+            src="./assets/imgs/unilever.png"
+            alt=""
+          />
+        </div>
+        <div class="">
+          <img
+            class="img-fluid partners"
+            src="./assets/imgs/cracker.png"
+            alt=""
+          />
+        </div>
+      </div>
+    </section>
+    <footer class="container-fluid last-section d-none d-sm-block d-md-block d-lg-block bg-light text-dark pt-5">
+      <div
+        class="container d-flex flex-row justify-content-center align-items-center gap-5"
+      >
+          <div class="logo-footer  pe-3">
+            <img
+              class="img-fluid"
+              src="./assets/imgs/mamas-kitchen.png"
+              alt=""
+            />
+          </div>
+        <div><p class="footer-text">2023 Mamas kitchen. Learn from the World’s Best Chefs. Some Right Reserve.</p></div>
+    </footer>
```

Comment:
- [ ] 🙏Please add padding to the bottom of the content in this section to match the original design. This adjustment will enhance the appearance 👩‍🦰of your website and ensure it resembles the original design.

Your design:
![image](https://github.com/Yonashenok/first-capstone-project/assets/63445083/6af45abd-00f4-4515-a5a7-ba09c165569e)

Original design:
![image](https://github.com/Yonashenok/first-capstone-project/assets/63445083/f52628be-6a27-406b-a906-c4c7d6de17a8)